docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss

```diff
@@ -0,0 +1,15 @@
+.node--type-multi-step-page {
+  .field__item {
+    @apply mb-5;
+  }
+}
+
+.nav-pills.non-sequential {
+  .nav-link {
+    &::before {
+      @extend %icon;
+      @include getIcon("chevron-right");
+      @apply absolute p-0 text-[2rem];
```

Comment:
Shouldn't we add 2rem font size to the tailwind config file?

Reply:
we use it only in 1 place